src/components/select-field/BaseSelectField.js

```diff
@@ -336,20 +337,21 @@ class BaseSelectField extends React.Component<Props, State> {
         }
     };
 
-    getItemFromFilteredOptions = (index: number) => {
+    getFilteredOptions = () => {
```

Comment:
Refactored this because renderSelectOptions method uses the filtering mechanism so I thought I'd have this method only handle the filtering, getting an element with the index can be done at a lower level.